import random
random.seed(2)

print(random.random())
print(random.random())
print(random.random())

# Producing error 
produce error
